homeassistant/components/light/flux_led.py

```diff
@@ -34,6 +34,8 @@
     vol.Optional(CONF_NAME): cv.string,
     vol.Optional(ATTR_MODE, default='rgbw'):
         vol.All(cv.string, vol.In(['rgbw', 'rgb'])),
+    vol.Optional(CONF_PROTOCOL, default=None):
+        vol.All(cv.string, vol.In(['ledenet'])),
```

Comment:
@bah2830 : You have to change the library to support lower case. https://github.com/Danielhiversen/flux_led/blob/master/flux_led/__init__.py#L642